frontends/web/src/i18n/i18n.ts

```diff
@@ -59,56 +82,49 @@ i18Init.init({
   },
 
   react: {
-    useSuspense : true, // Not using Suspense you will need to handle the not ready state yourself
+    useSuspense: true // Not using Suspense you will need to handle the not ready state yourself
   },
 
   backend: {
     projectId: locizeProjectID,
-    referenceLng: 'en'
-  },
+    referenceLng: defaultLang
+  }
 });
 
-i18n.addResourceBundle('ar', 'app', appTranslationsAR);
-i18n.addResourceBundle('cs', 'app', appTranslationsCS);
-i18n.addResourceBundle('de', 'app', appTranslationsDE);
-i18n.addResourceBundle('en', 'app', appTranslationsEN);
-i18n.addResourceBundle('fr', 'app', appTranslationsFR);
-i18n.addResourceBundle('ja', 'app', appTranslationsJA);
-i18n.addResourceBundle('ms', 'app', appTranslationsMS);
-i18n.addResourceBundle('nl', 'app', appTranslationsNL);
-i18n.addResourceBundle('ru', 'app', appTranslationsRU);
-i18n.addResourceBundle('pt', 'app', appTranslationsPT);
-i18n.addResourceBundle('hi', 'app', appTranslationsHI);
-i18n.addResourceBundle('bg', 'app', appTranslationsBG);
-i18n.addResourceBundle('tr', 'app', appTranslationsTR);
-i18n.addResourceBundle('zh', 'app', appTranslationsZH);
-i18n.addResourceBundle('fa', 'app', appTranslationsFA);
-i18n.addResourceBundle('es', 'app', appTranslationsES);
-i18n.addResourceBundle('sl', 'app', appTranslationsSL);
-i18n.addResourceBundle('he', 'app', appTranslationsHE);
-i18n.addResourceBundle('it', 'app', appTranslationsIT);
-
-i18n.on('languageChanged', (lng) => {
-  // Set userLanguage in config back to empty if system locale matches
-  // the newly selected language lng to make the app use native-locale again.
-  // This also covers partial matches. For example, if native locale is pt_BR
-  // and the app has only pt translation, assume they match.
-  //
-  // Since userLanguage is stored in the backend config as a string,
-  // setting it to null here in JS turns it into an empty string "" in Go backend.
-  // This is ok since we're just checking for a truthy value in the language detector.
-  return getNativeLocale().then((nativeLocale) => {
+// load the default language first
+loadLanguage(defaultLang);
```

Comment:
I found a small bug in the Qt app:

When the userLanguage in config.json is an empty string (or not defined) the Qt BBApp will always start with English instead of using the native language. 

We purposely store an empty string in case the user changes the language to be the same as the system.

How to test:
- change system to something else than English, i.e. German
- change userLanguage to empty string in `~/Library/Application\ Support/bitbox/config.json`
- open the Qt app, English loads, but expected is that the BBapp loads in German

I can't reproduce this well in webdev as native-locale gives me an empty string on webdev so it always goes to En for me in webdev.

Reply:
I played a bit around and the 2nd commit  in https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/3053 fixes the issue for me. Could you test with this and if it works for you squash it into your commit?

Reply:
Your version actually partly tries to load to German (with de system and userlang empty string) but the UI is still in English:
<img width="1121" alt="Bildschirmfoto 2024-11-15 um 12 18 39" src="https://github.com/user-attachments/assets/8df0151c-f785-4419-8441-9f48bf9704ea">